recipes/jwt-cpp/config.yml

```diff
@@ -1,7 +1,9 @@
 versions:
-    "0.3.1":
-        folder: all
-    "0.4.0":
-        folder: all
-    "0.5.0":
-        folder: all
+  "0.3.1":
```

Comment:
It would have been nice to have these also reverse ordered.

Reply:
Feature request for our favorite bot